webapp/graphite/intervals.py

```diff
@@ -19,6 +19,9 @@ def __repr__(self):
   def __iter__(self):
     return iter(self.intervals)
 
+  def __len__(self):
+    return len(self.intervals)
+
```

Comment:
@DanCech i'm trying to see where that's used, but i can't find it. Do we need that or can we drop it for the sake of reducing the size of the PR?